modules/github/Makefile.release

```diff
@@ -19,3 +19,11 @@ github/download-public-release:
 	$(call assert-set,VERSION)
 	$(call assert-set,OUTPUT)
 	$(BUILD_HARNESS_PATH)/bin/github_download_public_release.sh $(VERSION) $(OUTPUT)
+
+## Push all release artifacts to GitHub
+## To use ghr, you need a GitHub API token.
```

Comment:
Target annotations `##` must be on one-line and cannot repeat. Also, they must immediately precede the target name. This is due to how it's implemented with `awk`: https://github.com/cloudposse/build-harness/blob/master/Makefile.helpers#L39-L51

Instead, notes to developers should just use a single `#`. See other examples in makefiles.